packages/kotti-ui/source/kotti-card/types.ts

```diff
@@ -10,6 +10,12 @@ export namespace KottiCard {
 	export const propsSchema = z.object({
 		imgPosition: z.nativeEnum(ImagePosition).default(ImagePosition.TOP),
 		imgUrl: z.string().nullable().default(null),
+		isImgLoading: z.boolean().default(false),
+		isTextLoading: z.boolean().default(false),
```

Comment:
IS it really necessary to have 2 loadings instead of just 1?

Reply:
the reason why I created these 2 loading props is because of the requirements on how we want to show next the images/body text:

> For the catalog case, it becomes more important probably that the phases of loading are clarified as follow, depending on the time needed to load the parts thumbnails:

Card group (including metadata - e.g. part name, price, ID, etc.) is visualized in skeleton mode while loading.

If metadata for all cards is loaded before all preview images are loaded, the metadata is displayed for all cards as soon as the last piece of metadata loads. Images continue to load in skeleton

If all images load within the x period (a period of time, usually measured in seconds, that you need to specify depending on the use case), they are shown as soon as the last image loads.

If all images have not finished loading at the end of the x period, the loaded images are shown one by one, and the pending images are visualized in skeleton mode while still loading.

So, apparently we want potentially to show metadata before the image and keep the images as skeleton.